test/HTML5/Serializer/OutputRulesTest.php

```diff
@@ -645,8 +645,8 @@ public function testHandlingInvalidRawContent()
 
         $contents = $this->html5->saveHTML($dom);
 
-        $this->assertTrue(false !== strpos($contents, '<script id="template" type="x-tmpl-mustache">
+        $this->assertContains('<script id="template" type="x-tmpl-mustache">
```

Comment:
This should be `assertStringContainsString` (see https://phpunit.readthedocs.io/en/8.4/assertions.html#assertstringcontainsstring).